controllers/imagestream_reconciler.go

```diff
@@ -0,0 +1,58 @@
+package controllers
+
+import (
+	"context"
+	"fmt"
+
+	imagev1 "github.com/openshift/api/image/v1"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/filter"
+	"github.com/rh-ecosystem-edge/kernel-module-management/internal/syncronizedmap"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+//+kubebuilder:rbac:groups="core",resources=imagestreams,verbs=get;patch;list;watch
```

Comment:
I don't think we are patching the ImageStream?
```suggestion
//+kubebuilder:rbac:groups="core",resources=imagestreams,verbs=get;list;watch
```

Reply:
Yea we don't patch, I will remove it.

Reply:
Removed.